sdk/lib/crt/printf/streamout.c

```diff
@@ -699,6 +1026,13 @@ streamout(FILE *stream, const TCHAR *format, va_list argptr)
             }
         }
 
+        /* Optional exponent */
+        const TCHAR* pexp = exp_string;
+        while(*pexp != 0)
```

Comment:
```suggestion
        while (*pexp != 0)
```
I wouldn't give a fuck about this personally. But I know, that others will at some point in the future. So let's do it right now to avoid follow-ups. ;-)

Reply:
That piece of code is in fact obsolete. Will remove it.